web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -239,6 +244,7 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
 
             const jobProps: JobVariables = {
                 name: jobName,
+                revisions: revisionId ? [revisionId] : revisions,
```

Comment:
revisions in response from back are always list of revisions, do we need to cover such cases?

Reply:
it is case when I get revisionId from url params, when clicking at new job in document